src/LeagueWrap/StaticOptimizer.php

```diff
@@ -73,6 +73,9 @@ public function setStaticInfo(Staticdata $staticData)
 			if (count($ids) > 1)
 			{
 				// group up the calls
+				if ( substr($method, -1) == 'y'){ // if last char is 'y' replace for plural
```

Comment:
Changing `-y` to `-ies` only applies when the `y` is not preceded by a vowel. Examples: `key` becomes `keys`; `boy` becomes `boys`.

Perhaps a regular expression would be more appropriate:

```php
if (preg_match('/(?<![aeiou])y$/', $method))
```